Makefile.third

```diff
@@ -682,7 +682,7 @@ $(LUA_SPORE_ROCK): $(THIRDPARTY_DIR)/lua-Spore/*.*
 		$(CMAKE) $(CMAKE_FLAGS) -DOUTPUT_DIR="$(CURDIR)/$(OUTPUT_DIR)" \
 		-DLUA_SPORE_VER=$(LUA_SPORE_VER) -DLD="$(CC)" \
 		-DCC="$(CC)" -DCFLAGS="$(CFLAGS) -I$(LUAJIT_DIR)/src" \
-		-DLUAROCKS=$(if $(DARWINHOST),"luarocks --lua-dir=/usr/local/opt/lua@5.1",luarocks) \
+		-DLUAROCKS=$(if $(DARWINHOST),"luarocks --lua-dir=/usr/local/opt/lua@5.1","luarocks --lua-version=5.1") \
```

Comment:
That's 3.x only, IIRC.

(For ref., I still patch my local copy w/ http://trac.ak-team.com/trac/browser/niluje/Configs/trunk/Kindle/Misc/koreader-luarocks-3.patch because I'm way too lazy to figure out what's got its panties in a twist ;)).